### PR TITLE
CXTB-557 ESCarePartner Platform - Call Participants Device And Complete Call

### PR DESCRIPTION
**Pull request by dguzmanneural**
-Added ES test
-Added CH test
-Added tests to set

image
image

### CODE REVIEW

File: tests/cases/es_care_partner_platform/case_es_care_platform_calls.yaml [hunk 1]
```diff
@@ -220,4 +220,245 @@ TestESCarePartnerBrowsingWhileOnCall:
...
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: CLASS
+          Value: 'is-open'
+  #Events and Calendar
```
Comment (csanchez-nts) [inline tests/cases/es_care_partner_platform/case_es_care_platform_calls.yaml:394]: Extra tab needed

File: tests/cases/es_care_partner_platform/case_es_care_platform_calls.yaml [hunk 2]
```diff
@@ -220,4 +220,245 @@ TestESCarePartnerBrowsingWhileOnCall:
...
+    #Check Calendar
+    - Type: click
+      Strategy: xpath
+      Role: desktopChrome
+      Id: $PAGE.care_platform_call_portal.calendar_any_day_without_event$
+    - Type: wait_for
+      Strategy: xpath
+      Role: desktopChrome
+      Id: $PAGE.care_platform_call_portal.calendar_no_events_message$
+  #End Call
```
Comment (csanchez-nts) [inline tests/cases/es_care_partner_platform/case_es_care_platform_calls.yaml:421]: same here

File: tests/cases/es_care_partner_platform/case_ch_care_platform_calls.yaml [hunk 3]
```diff
@@ -220,4 +220,245 @@ TestCHCarePartnerBrowsingWhileOnCall:
...
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: CLASS
+          Value: 'is-open'
+  #Events and Calendar
```
Comment (csanchez-nts) [inline tests/cases/es_care_partner_platform/case_ch_care_platform_calls.yaml:394]: One more tab needed

**Review by csanchez-nts — COMMENTED** [review]
Format comments to have same indentation

**Review by csanchez-nts — APPROVED** [review]

**Review by miguel-hquez — APPROVED** [review]

**Pull request merged**